resolversrv/models.py

```diff
@@ -0,0 +1,91 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+
+from flask import current_app
+from sqlalchemy import Integer, String, Column, and_
+from sqlalchemy.dialects.postgresql import ARRAY
+from sqlalchemy.ext.declarative import declarative_base
+from sqlalchemy.orm.exc import NoResultFound
+
+
+Base = declarative_base()
+
+class DataLinks(Base):
+    __tablename__ = 'datalinks'
+    __table_args__ = ({"schema": "nonbib"})
```

Comment:
what is __table_args__ used for? we should avoid specifiying the target schema (db)

Reply:
that is the way the Postgres db is setup. you remove table_args it would not find the schema. but this shall go away once the db in AWS, right?

Reply:
I don't think this is the right way to go, but since I know eventually this goes away, I altered the database and set to always use nonbib schema and so I am able to remove it from code. Test is happy too.